utils/artifacts.py

```diff
@@ -53,10 +51,16 @@ def get_release_and_version(package_path: str) -> Tuple[bool, str, str, str, str
     """
     Load version and release info from deepsparse package
     """
+    # deepsparse/src/deepsparse/version.py always exists, default source of truth
+    version_path = os.path.join(package_path, "version.py")
 
-    version_path = os.path.join(package_path, "generated_version.py")
-    if not os.path.exists(version_path):
-        version_path = os.path.join(package_path, "version.py")
+    # If deepsparse/engine-version.txt exists, use the generated_version
+    engine_version_path = os.path.abspath(
+        os.path.join(package_path, os.pardir, os.pardir, "engine-version.txt")
```

Comment:
Do we not rely on `version.py` at all going forward?

Reply:
We actually use `version.py` more often now after this change. The only way to get `deepsparse/engine-version.txt` is if you're working on the internal engine interface, which is how you get the `generated_version.py` as well (or through the artifact store).
Another approach to this would be to not gather `generated_version.py` from the artifact at all.. I'll need to make sure this doesn't break anything, might be better than this method

Reply:
`generated_version.py` is needed for other functions of deepsparse to work, so not a valid solution